core/multiproc/inc/TProcPool.h

```diff
@@ -98,11 +89,11 @@ class TProcPool : private TMPClient {
       kNoTask = 0,   ///< no task is being executed
       kMap,          ///< a Map method with no arguments is being executed
       kMapWithArg,   ///< a Map method with arguments is being executed
-      kMapRed,       ///< a MapReduce method with no arguments is being executed
-      kMapRedWithArg, ///< a MapReduce method with arguments is being executed
       kProcByRange,   ///< a ProcTree method is being executed and each worker will process a certain range of each file
       kProcByFile,    ///< a ProcTree method is being executed and each worker will process a different file
-   } fTask; ///< the kind of task that is being executed, if any
+   };
+
+   ETask fTask = ETask::kNoTask; ///< the kind of task that is being executed, if any
```

Comment:
This data member seems misnamed ... shouldn't it be more like fTaskType?  (i.e. it is not the task itself but a description of the (kind of) task ... or so it seems.